pkg/helm/release/manager.go

```diff
@@ -22,6 +22,8 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/operator-framework/operator-sdk/pkg/helm/internal/types"
```

Comment:
Nit: Move this import down to a separate section (but keep jsonpatch here). We try to keep our imports arranged in three groups (standard library, external, internal)